### PR TITLE
Add a wiki page explaining how to quickly load data into the development server.

### LINKED ISSUE #30: Add a wiki page explaining how to quickly load data into the development server.
For solving certain issues, developers may need to populate the local server datastore with a bunch of skills/topics/lessons/users/etc. I keep seeing new developers asking questions on how to do this (e.g. how to get a translation into their local contributor dashboard page so that they can try reviewing it, as a first step towards fixing a bug in the review process).

We should add instructions for how to do this, so that issue reproduction instructions can point to them. It would be nice to define (and implement buttons in the /admin Activities tab for) a set of "standard configurations" that we maintain for this purpose -- these configurations can cover different types of data (e.g. LaCE-related vs contributor-dashboard related) and be optimized for different things (e.g. multiple classrooms vs 1000 items for load testing vs single copy of each entity).

### PR DESCRIPTION
**Pull request by chris7716**
Fixes #30

### CODE REVIEW

File: How-To-Load-Data.md
Comment (gp201) [inline How-To-Load-Data.md:1]: Please add a Table of contents to the beginning of the file

File: How-To-Load-Data.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### Populating translation opportunities
+- Start a local server and sign in as an administrator (testadmin@example.com, assume username “a”).
+- Navigate to the Admin page.
+- Click the Roles tab.
+- In the Role editor, enter the admin username (“a”). Click Add Role, and add the “Curriculum Admin” role.
+- Navigate to the Activities tab. Click Load Data under Load dummy new structures data.
+- You should now see translation opportunities in the Translate Text tab at /contributor-dashboard.
```
Comment (gp201) [inline How-To-Load-Data.md]: If these are steps convert them to a numbered list.

**Review by gp201 — CHANGES_REQUESTED** [review]
Thanks, @chris7716 ! I left some comments based on the work you have done so far. PTAL

File: How-To-Load-Data.md (hunk 1 above)
Comment (chris7716) [reply How-To-Load-Data.md]: Done

File: How-To-Load-Data.md
Comment (chris7716) [reply How-To-Load-Data.md:1]: Done

**Comment by U8NWXD** [comment]
It looks like this is ready for a re-review, so assigning @gp201. In the future, please remember to assign reviewers when you want them to take a look @chris7716

File: How-To-Load-Data.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Table of contents
+- [Populating translation opportunities](#populating-translation-opportunities)
+- [Populating question opportunities](#populating-question-opportunities)
```
Comment (gp201) [inline How-To-Load-Data.md:3]: Add a section on how to populate explorations and classroom also

**Review by gp201 — CHANGES_REQUESTED** [review]

**Comment by U8NWXD** [comment]
@chris7716 just a friendly reminder to take a look at gp201's comments on this PR

**Comment by Shivkant-Chauhan** [comment]
@chris7716 friendly ping on this! can you please tell an expected merge date for this PR?

**Comment by U8NWXD** [comment]
I'll take over this PR and get it merged since @chris7716 doesn't appear to be active on it

**Comment by SD-13** [comment]
As discussed in the meeting, I will resurrect this PR soon.

File: How-To-Load-Data.md (hunk 2 above)
Comment (chris7716) [reply How-To-Load-Data.md:3]: Done

**Comment by chris7716** [comment]
@U8NWXD @gp201 sorry for being inactive on this. I was away since I was having COVID. I have made the requested changes. Could PTAL? Thanks!

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**